packages/components/src/button/src/CrossButton.css

```diff
@@ -0,0 +1,9 @@
+.o-ui-button-sm.o-ui-icon-button.o-ui-cross-button .o-ui-button-icon {
+    width: 1.25rem;
+    height: 1.25rem;
+}
+
+.o-ui-button-md.o-ui-icon-button.o-ui-cross-button .o-ui-button-icon {
```

Comment:
buttons can have 4 sizes, should they all be here? 

"xs" | "sm" | "md" | "2xs"